currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -18,6 +18,12 @@ class ResCurrencyRateProviderECB(models.Model):
         selection_add=[('ECB', 'European Central Bank')],
     )
 
+    @api.model
+    def _get_close_date(self):
+        if self.service == 'ECB':
+            return 17
```

Comment:
Close date 17? :)

Reply:
It should be `_get_close_time`?

Reply:
BCE Provider define 16:00 (https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html) but us define 17:00 (to ensure that data already exists)

Reply:
At least yes, and how to do you propose to use this method for providers like XE.com or OXR?